public/static/app/common/constants.js

```diff
@@ -144,7 +144,8 @@ mciModule
   .constant('CANARY_EXCLUSION_REGEX', /^((?!canary_|fio_|iperf|NetworkBandwidth).*)$/)
   .constant('CEDAR_APP_URL', 'https://cedar.mongodb.com')
   .constant("PERFORMANCE_ANALYSIS_AND_TRIAGE_API", {
-    BASE:'https://signal-processing-service.server-tig.prod.corp.mongodb.com',
+    BASE:'https://signal-processing-service.server-tig.staging.corp.mongodb.com',
```

Comment:
Was this change intentional? This is what will be in production evergreen

Reply:
Whoops, fixed